Add private hosting + polish (#314)
chainstack/docs: docs/platform/deploy-a-consortium-network.md

Chainstack supports [cloud](/glossary/cloud) and [hybrid](/glossary/hybrid) deployment of consortium blockchain networks.

## Deploy a Hyperledger Fabric network

1. Select a [consortium project](/glossary/consortium-project) and click **Get started** or **Create network**.
1. Provide **Network name**.
1. Under **Blockchain protocol**, select **Hyperledger Fabric**.
1. The consensus algorithm is [Raft](/blockchains/fabric#consensus). Click **Next**.

> 1. Under **Identity issuer**, select **Cryptogen**.

Author: It's in a different PR.

Chainstack supports [cloud](/glossary/cloud) and [hybrid](/glossary/hybrid) deployment of consortium blockchain networks.

> Deployment of multicloud and hybrid hosting networks is available starting from the Enterprise <a href="https://chainstack.com/pricing/" target="_blank">subscription plan</a>.

Reviewer: We should be consistent with terminology on the website, in the platform, in the docs

```suggestion
Deployment of cross-cloud, cross-region and hybrid networks is available starting from the Enterprise <a href="https://chainstack.com/pricing/" target="_blank">subscription plan</a>.
```

## Deploy a Hyperledger Fabric network

1. Select a [consortium project](/glossary/consortium-project) and click **Get started** or **Create network**.
1. Provide **Network name**.
1. Under **Blockchain protocol**, select **Hyperledger Fabric**.
1. The consensus algorithm is [Raft](/blockchains/fabric#consensus). Click **Next**.
1. Under **Identity issuer**, select or add an identity issuer. See [Manage your organization's identities](/platform/manage-your-organizations-identities).
1. Under **Hosting**, select **Chainstack** or **Private**. See [Support hosting options](/platform/supported-hosting-options).

> 	* For Chainstack hosting, select a cloud provider.

Reviewer: Here and below IMO should be

```suggestion
	* For Chainstack hosting, select a cloud provider and a region.
```

> 	* For private hosting, select an integration. See [Manage your organization's integrations](/platform/manage-your-organizations-integrations).

Reviewer 2: ```suggestion
	* For private hosting, select or add an integration. See [Manage your organization's integrations](/platform/manage-your-organizations-integrations).
```
